azure-mgmt-graph-rbac/src/main/java/com/microsoft/azure/management/graphrbac/ServicePrincipal.java

```diff
@@ -59,6 +60,7 @@
     /**
      * Container interface for all the definitions that need to be implemented.
      */
+    @Beta(SinceVersion.V1_1_0)
```

Comment:
Why do we have these Beta annotations here?  From what I can see, we already shipped ServicePrincipal(s) as interfaces in v1.0.0 in Beta. If that's the case but individual withers got added or changed in v1.1, then we should mark only those individual withers as `@Beta(SinceVersion.V1_1_0)`, not the Definition/Update fluent interfaces they are part of.